src/acl/Node.h

```diff
@@ -213,7 +211,6 @@ class acl_proxy_auth_match_cache
 
 /// \ingroup ACLAPI
 /// XXX: find a way to remove or at least use a refcounted ACL pointer
-extern const char *AclMatchedName;  /* NULL */
-
-#endif /* SQUID_ACL_H */
+extern const char *AclMatchedName;  /* nullptr */
 
+#endif /* SQUID_ACL_NODE_H */
```

Comment:
What if we just turned this on its head and moved to `#pragma once` ? 
It has been reportedly [supported by everyone](https://en.wikipedia.org/wiki/Pragma_once#Portability) for about 10 years, and would just make this whole point moot

Reply:
According to the [page](https://en.wikipedia.org/wiki/Pragma_once) you referenced, non-standard `#pragma once` comes with several significant caveats (besides the obvious lack of standardized universal support). Some of those caveats might be applicable to some Squid builds.

Include guards (i.e. the standard approach we have been using) seem to be an overall better alternative. I want to believe that we can agree on how to name new/changed ones -- it seems such a simple and non-controversial task! Right now, I do not see any significant signs that we cannot.

Reply:
yes, but.

In theory, there is no snandardised support. In practice all compilers support it.
In theory, there can be issues with symbolic links, in practice, we don't use symbolic links in our build chain.
Also, in theory it's easy to come up with consensus, in practice we're spending time on this

Let me see if I can come up with something automated as part of source-formatting, this could prove a viable alternate.

Reply:
Okay; PR #1630 can address this topic, hopefully for good, by automatically enforcing a standard